src/dualKpiSettingsModel.ts

```diff
@@ -0,0 +1,537 @@
+import powerbi from 'powerbi-visuals-api';
```

Comment:
I don't think that this is a good idea to make a one file with 500(only for now) lines of code just for settings. It's easier to maintain the code when it's separated into logic parts.

Reply:
You're right, the file grew bigger and bigger and surprisingly it reached 500 lines. I split the settings into dedicated files for each card.

Reply:
Totally agree. Also I want to suggest this PR to make settings imports more simple to maintain: [https://github.com/adiletelf/powerbi-visuals-dualkpi/pull/2](https://github.com/adiletelf/powerbi-visuals-dualkpi/pull/2)

Reply:
I looked at the PR. It looks much better, thanks)